string action_type

# Goal pose
geometry_msgs/PoseStamped goal_pose

# Joint goal
float64[] joint_deltas  # an array of doubles

# Goal as a named target
string named_target

---
# Fraction of successfully computed Cartesian path
float32 cartesian_fraction
